files/build_templates/buffers_config.j2

```diff
@@ -93,7 +94,14 @@ def
                 {{ default_cable }}
             {%- endif %}
         {%- else -%}
-            {{ default_cable }}
+            {%- if port_name.startswith('Ethernet-BP') %}
+                {%- if 'backplane' not in ports2cable %}
+                    {%- set _ = ports2cable.update({'backplane': '1m'}) %}
+                {%- endif -%}
+                {{ ports2cable['backplane'] }}
```

Comment:
we already have convention of `internal` ports define for multi-asic switch . Let's reuse it and not define another notion of `backplane`. We can resuse below logic .

  ```
  {%- set roles1 = 'internal' %}
                         {%- if 'internal' not in ports2cable %}
                             {%- set _ = ports2cable.update({'internal': '5m'}) %}
                         {%- endif -%}
```

Reply:
Previously we were using 5m for internal links. Do we need to use 1m  ?

Reply:
I set the default cable length of internal links to 5m.
The cable length can be overwritten in vendor buffer config template.